app/views/admin/attachments/index.html.erb

```diff
@@ -1,11 +1,20 @@
 <% page_title "Attachments for #{attachment.attachable_model_name}" %>
 <% page_class "attachments index" %>
+<% is_publication = attachment.attachable_model_name == 'publication' %>
 
 <div class="row">
   <section class="col-md-8">
     <h1>Attachments for <%= attachment.attachable_model_name %></h1>
 
     <%= attachable_editing_tabs(attachable) do %>
+      <p class="qa-helper-copy">
+        <strong>Note:</strong>
+        <% if is_publication %>
+          This is a Publication, so attachments will appear automatically.
+        <% else %>
+          This isn't a Publication, so attachments need to be inlined into the main body markdown to appear.
```

Comment:
_inlined_ – I'd say "referenced" or "included" or double check with a copywriter.
"Attachments need to be referenced in the body markdown to appear in your document"
"Attachments added to a publication will appear automatically"